README.md

```diff
@@ -0,0 +1,26 @@
+## Demo of "How to handle deprecated methods between versions"
+
+## About
+
+This module demonstrates how you can handle deprecated code between a few versions of the PrestaShop software.
```

Comment:
After your demo, we could add a link to the Youtube replay. The link would say "Deprecations are necessary for a software to move forward, you can watch our YouTube demo about this topic to learn more"